docs/core/tools/dotnet-workload-install.md

```diff
@@ -0,0 +1,106 @@
+---
+title: dotnet workload install command
+description: The 'dotnet workload install' command installs optional workloads.
+ms.date: 07/08/2021
+---
+# dotnet workload install
+
+**This article applies to:** ✔️ .NET 6 Preview SDK and later versions
```

Comment:
Do you want to specify which preview?

Reply:
>Do you want to specify which preview?

I'm thinking that specifying preview number here where it applies to the whole article wouldn't be helpful because parts of the feature were added in 4, 5, and 6, and 6 is the current preview release. An exception is --source, which won't be there until 7, so I'll add a note to that effect for that option.